x-pack/plugins/ml/public/components/field_type_icon/field_type_icon.js

```diff
@@ -12,43 +12,68 @@ import { EuiToolTip } from '@elastic/eui';
 // don't use something like plugins/ml/../common
 // because it won't work with the jest tests
 import { ML_JOB_FIELD_TYPES } from '../../../common/constants/field_types';
+import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
 
-export function FieldTypeIcon({ tooltipEnabled = false, type }) {
+export const FieldTypeIcon = injectI18n(function FieldTypeIcon({ tooltipEnabled = false, type, intl }) {
   let ariaLabel = '';
   let iconClass = '';
   let iconChar = '';
 
   switch (type) {
     case ML_JOB_FIELD_TYPES.BOOLEAN:
-      ariaLabel = 'boolean type';
+      ariaLabel = intl.formatMessage({
+        id: 'xpack.ml.fieldTypeIcon.booleanTypeAriaLabel',
+        defaultMessage: 'boolean type'
+      });
       iconClass = 'fa-adjust';
       break;
     case ML_JOB_FIELD_TYPES.DATE:
-      ariaLabel = 'date type';
+      ariaLabel = intl.formatMessage({
+        id: 'xpack.ml.fieldTypeIcon.dateTypeAriaLabel',
+        defaultMessage: 'date type'
+      });
       iconClass = 'fa-clock-o';
       break;
     case ML_JOB_FIELD_TYPES.NUMBER:
-      ariaLabel = 'number type';
+      ariaLabel = intl.formatMessage({
+        id: 'xpack.ml.fieldTypeIcon.numberTypeAriaLabel',
+        defaultMessage: 'number type'
+      });
       iconChar = '#';
       break;
     case ML_JOB_FIELD_TYPES.GEO_POINT:
-      ariaLabel = 'geo_point type';
+      ariaLabel = intl.formatMessage({
+        id: 'xpack.ml.fieldTypeIcon.geoPointTypeAriaLabel',
+        defaultMessage: '{geoPointParam} type'
+      }, { geoPointParam: 'geo_point' });
```

Comment:
@Bamieh, hello, could you please confirm that "geo point" string should not be translated? Otherwise, could you clarify why it is passed in as a parameter?